nav2_ros_common/include/nav2_ros_common/validate_messages.hpp

```diff
@@ -65,6 +68,11 @@ bool validateMsg(const std::array<double, N> & msg)
    */
   for (const auto & element : msg) {
     if (!validateMsg(element)) {return false;}
+
+    if (std::abs(element) > MAX_COVARIANCE || std::abs(element) < MIN_COVARIANCE) {
+      // Reject extreme values (security: prevent DoS attacks)
```

Comment:
How would this be a DoS? I think this should just mention checking for covar valid range